shuffle/cli/context.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
What is the difference between this file and the examples/main/context.ts?

Reply:
botched the rename. I'll delete this one.